concert/ext/tangoservers/reco.py

```diff
@@ -49,6 +50,14 @@ def set_{0}(self, values):
 """
 
 
+async def read_images(path):
+    with TiffSequenceReader(path) as reader:
+        image_stack = []
+        async for image in reader.read_range():
+            image_stack.append(image)
+        return image_stack
```

Comment:
This is doubled now in the local addons and here. How about putting it somewhere like `concert.readers` and using that _one_ function on both positions?

Reply:
This is part of the commit that I would drop for now and come up with a more elegant solution.